content/blog/2016/2016-11-21-serverless-ci-hyper-sh-jenkins-plugin.adoc

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: "Serverless CI with the new Hyper.sh plugin for Jenkins"
+tags:
+- jenkins
+- docker
+- containers
+- serverless
+- immutable servers
+- CI
+- CD
+author: jmknyc06
+---
+
+= Serverless CI
+
+== What is Hyper.sh?
+
+https://hyper.sh/docker-hosting[Hyper.sh] is a container-native cloud service. Different from traditional IaaS and CaaS providers, Hyper_ does not employ a VM cluster. Instead containers are the only object the users work with.
```

Comment:
I don't think "IaaS" and "CaaS" are not supremely well known acronyms, so it might be helpful to link them to definition or wikipedia pages?